scripts/ci.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -e
+
+setupNvm() {
+  if ! test -e "$NVM_DIR/nvm.sh"; then
+    echo -e "NVM not found. NVM is required to run this project."
+    exit 1
+  else
+    source "$NVM_DIR/nvm.sh"
+    nvm install
+  fi
+}
+
+buildJs() {
+  echo "##teamcity[compilationStarted compiler='webpack']"
```

Comment:
this creates a group (?) in the build log

![image](https://user-images.githubusercontent.com/836140/81948026-28ac5780-95f9-11ea-816f-0e1fa3290b57.png)